### PR TITLE
Prepare release 3.0.0-exp.2

### PR DESCRIPTION
**Pull request by judubu**
## Purpose of this PR

**Ticket/Jira #:** https://jira.unity3d.com/browse/USDU-197

Update changelog, bumped package version and changed yamato config to use 2021.2 instead of 2021.1

### CODE REVIEW

File: package/com.unity.formats.usd/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changes in usd-unitysdk for Unity
 
+## [3.0.0-exp.2] - 2021-09-29
+### Features
+- All interpolation types are now properly supported for Mesh standard attributes (normals, tangents, uvs, colors).
+
+### Bug Fixes
+- Fixed the import of facevarying UVs which showed seams on Meshes.
+- Fixed an import bug causing abstract primitives to be loaded as Game Objects.
+- Fixed the Alembic support.
```
Comment (mfe) [inline package/com.unity.formats.usd/CHANGELOG.md]: Fixed the broken Alembic import ?

Comment (judubu) [reply package/com.unity.formats.usd/CHANGELOG.md]: yep, I can do that

File: package/com.unity.formats.usd/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Changes in usd-unitysdk for Unity
 
+## [3.0.0-exp.2] - 2021-09-29
+### Features
+- All interpolation types are now properly supported for Mesh standard attributes (normals, tangents, uvs, colors).
+
+### Bug Fixes
+- Fixed the import of facevarying UVs which showed seams on Meshes.
```
Comment (mfe) [inline package/com.unity.formats.usd/CHANGELOG.md:8]: Fixed the import of facevarying UVs containing seams ?
 The current sentence does not look very english.

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested with:
- 2021.2.0b12 (Windows, MacOS)

File: package/com.unity.formats.usd/CHANGELOG.md (hunk 2 above)
Comment (mfe) [reply package/com.unity.formats.usd/CHANGELOG.md:8]: Fixed the import of facevarying UVs that was broken for Meshes with seams ?

**Pull request merged**